app/src/main/java/com/codesoom/assignment/App.java

```diff
@@ -1,10 +1,33 @@
 package com.codesoom.assignment;
 
+import com.sun.net.httpserver.HttpHandler;
+import com.sun.net.httpserver.HttpServer;
+
+import java.io.IOException;
+import java.net.InetSocketAddress;
+
+/*
+* 클라이언트의 요청을 받아서 처리할
+* HttpServer 생성
+* */
```

Comment:
Javadoc을 작성해 주셨네요
```suggestion
/*
* 클라이언트의 요청을 받아서 처리할
* HttpServer 생성
*/
```

요렇게 마무리 해주시면 좋을 것 같습니다